app/uk/gov/hmrc/tariffclassificationfrontend/controllers/CommonController.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tariffclassificationfrontend.controllers
+
+import play.api.Logger
+import play.api.mvc.Result
+import uk.gov.hmrc.play.bootstrap.controller.FrontendController
+import uk.gov.hmrc.tariffclassificationfrontend.model.{ErrorCode, JsErrorResponse}
+
+trait CommonController extends FrontendController {
```

Comment:
There is an implicit `hc` (header carrier) in `FrontendController` .
We should use it and pass it from the controllers all the way down to the connectors.